mantis-runtime-loader/src/main/java/io/mantisrx/runtime/loader/config/WorkerConfiguration.java

```diff
@@ -96,7 +96,7 @@ default Time getHeartbeatInterval() {
     // ------------------------------------------------------------------------
     //  RPC related configurations
     // ------------------------------------------------------------------------
-    @Config("mantis.taskexecutor.rpc.external-address")
+    @Config({"mantis.taskexecutor.rpc.external-address", "HOSTNAME"})
```

Comment:
maybe use `MANTIS_TASKEXECUTOR_RPC_EXTERNAL_ADDRESS` to prevent unintentionally using a value. Same for the other configs. Using the same config name should also reduce confusion.

Reply:
this one is a bit tricky as the HOSTNAME is dynamic but the agent needs it at startup (so in test I cannot get this value till it's started but the agent needs it to start entry point). 
Let me add "MANTIS_TASKEXECUTOR_RPC_EXTERNAL_ADDRESS" in front of "HOSTNAME" and keep it as a last fallback option.

Reply:
updated both